### PR TITLE
Add Spheron in the cloud infrastructure section inside linea tooling

### PR DESCRIPTION
**Pull request by rekpero**

### CODE REVIEW

File: project-words.txt [hunk 1]
```diff
@@ -789,3 +789,7 @@ zoomable
 zpao
 zpaoAave
 zpaoAaveAave
```
Comment (emilyJLin95) [inline project-words.txt:791]: Can you put this in alphabetical order?

It technically doesn't matter, but it helps with merge conflicts when multiple people are editing this file

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]
@rekpero one comment!

File: project-words.txt (hunk 1 above)
Comment (rekpero) [reply project-words.txt]: Check now, updated the order

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**